MatrixSDK/Data/MXRoom.m

```diff
@@ -3348,8 +3348,26 @@ - (void)acknowledgeEvent:(MXEvent*)event andUpdateReadMarker:(BOOL)updateReadMar
     }
 }
 
+- (void)setUnread
+{
+    [mxSession.store setUnreadMarkerForRoom:self.roomId];
+    [mxSession.store commit];
```

Comment:
Probably won't be an issue in practice (`store` is always `MXFileStore`), but `commit` is marked as `optional` on the protocol, so should be surrounded with `if ([mxSession.store respondsToSelector:@selector(commit)])` for safety as is done elsewhere in that file